contracts/interfaces/IPool.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-3.0-only
 
-pragma solidity >=0.5.0;
+pragma solidity >=0.8.9;
```

Comment:
Use the minimum version in interfaces (`>=0.8.0`) so they're usable by other contracts/projects with a lower patch version (ex. `0.8.4`).